eng/pipelines/installer/jobs/osx-build.yml

```diff
@@ -1,38 +1,25 @@
 parameters:
   name: ''
+  platform: ''
+  pool: ''
 
 jobs:
-- job: ${{ parameters.name }}
-  pool:
-    ${{ if eq(variables['System.TeamProject'], 'public') }}:
-      name: Hosted macOS
-    ${{ if ne(variables['System.TeamProject'], 'public') }}:
-      name: Hosted Mac internal
-  strategy:
-    matrix: 
-      debug:
-        _BuildConfig: Debug
-      release:
-        _BuildConfig: Release
-  workspace:
-    clean: all
-  variables: 
-    CommonMSBuildArgs: >-
-      /p:Configuration=$(_BuildConfig)
-      /p:PortableBuild=true
-  steps:
+- template: base-job.yml
+  parameters:
+    ${{ insert }}: ${{ parameters }}
 
-  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-    - task: NuGetAuthenticate@0
+    buildVariables:
+      CommonMSBuildArgs: >-
```

Comment:
Just to learn, what does: `>-` do?

Reply:
I'm seeing this for the first time too...

Reply:
`>` replaces newlines with spaces (vs. `|` which keeps newlines), `-` strips the newline at the end. http://yaml-multiline.info/ is a nice resource for this, and it has links to the spec.

Reply:
Thanks, pretty useful 😄